Source_Code/Server/ClueLessServer.java

```diff
@@ -467,6 +478,31 @@ else if ( ! ( possibleRefutations.getWeapons().isEmpty()) )
         }
     }
 
+    public void processAccuseRequest( AccuseRequest acccuseRequest )
+    {
+        // prep the AccuseNotification here with who and what they are guessing.
+        AccuseNotification accuseNotification = new AccuseNotification(acccuseRequest.PlayerName, acccuseRequest.AccuseHand);
+
+        // Check to see if the person is correct
+        if (EnvelopeHand.isEqual(acccuseRequest.AccuseHand))
+        {
+            // If correct, tell everyone, end game
+            accuseNotification.setCorrect(true);
+            sendToAllPlayers( accuseNotification );
+            sendToAllPlayers( new GameStart(false) );
+            ServState = ServerState.Lobby;
+        }
+        else
+        {
+            // If incorrect. tell everyone, set player "out"
+            accuseNotification.setCorrect(false);
+            // TODO - looks like setting this to false doesn't actually prevent them from making moves
```

Comment:
ClueLessServer:nextMove will auto-skip disconnected and out players when assigning a turn. Need to do more testing to be sure it works 100%, but it seems to at least work at a rudimentary level. Need to add the turn tracking to the processAccuse, and I think everything'll work as expected (I can do this, just want to put the status here)

Reply:
comment made before testing. Will take out now, I am sure we will notice if this does not work